infrastructure/run_ansible_vagrant

```diff
@@ -69,5 +69,6 @@ function buildMigrations() {
 buildMigrations
 
 ansible-playbook -D -v \
-     ansible/site.yml \
+    "$@" \
```

Comment:
Does this work when it's inside quotes?

Reply:
I would expect for it to be break, if it were to break, on any token containing a space. The `-t <tag>` param worked, I expect it should work always. I have not tested with many args though. The other element, this was modeled after  what we have in run_deployment, I don't remember exactly if multiple args were tested there as well (I think so, but my recollection is fuzzy).

```
ansible-playbook \
  --vault-password-file "$VAULT_PASSWORD_FILE" \
 "$@" \
 ansible/site.yml
```